src/main/java/org/broadinstitute/consent/http/resources/LibraryCardResource.java

```diff
@@ -81,6 +83,13 @@ public Response createLibraryCard(@Auth AuthUser authUser, String libraryCard) {
     try{
       User user = userService.findUserByEmail(authUser.getName());
       LibraryCard payload = new Gson().fromJson(libraryCard, LibraryCard.class);
+      User lcUser = userService.findUserByEmail(payload.getUserEmail());
+
+      if (Objects.isNull(lcUser)) {
+        throw new BadRequestException("User email does not belong to a valid user.");
+      }
```

Comment:
The service call will throw a NFE if the user is null, so this check should never happen.

Reply:
Removed the unecessary thrown exception.